### ISSUE TITLE
Android 14 vs BATTERY_STATS

### ISSUE THREAD

**Issue by dreimer1986** [asksven/BetterBatteryStats#915]
Pixel 7 Pro here. Running Beta3.

![Screenshot_20230610-105601](https://github.com/asksven/BetterBatteryStats/assets/29841368/150cc716-e70f-4b95-b19c-543c047c9ef8)

**Comment by adrianmmiller** [comment]
yup me too...suspect Sven hasnt gotten to A14 compatibility yet...

**Comment by clayissimo** [comment]
> yup me too...suspect Sven hasnt gotten to A14 compatibility yet...

Is BBS still being developed (incl. bug fixes) by Sven?

**Comment by jason-eu** [comment]
![Screenshot_20231020-150115_Brave - Beta.png](https://github.com/asksven/BetterBatteryStats/assets/13890252/a79c8dfc-e740-44bf-8039-abba66bc3369)

[BetterBatteryStats xda edition for Android by asksven](https://install.appcenter.ms/users/sven-knispel-g51w/apps/betterbatterystats-xda-edition/distribution_groups/testers)

**Comment by p3dr01978** [comment]
Good morning, any news on Android 14 update? many thanks

**Comment by TitanGames-729** [comment]
Same here, Android 14 with magisk and it won't work. I can get some raw stats but still it's not enough.
